frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -84,6 +86,55 @@ export default class Metadata extends React.Component<Props> {
         </Localized>;
     }
 
+    renderGroupComment(entity: Entity): React.Node {
+        if (!entity.group_comment) {
+            return null;
+        }
+
+        let comment = entity.group_comment;
+
+        return <Localized id='entitydetails-Metadata--group_comment' attrs={ { title: true } }>
+            <Property title='Group Comment' className='comment'>
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { comment }
+                </Linkify>
+            </Property>
+        </Localized>;
+    }
+
+    state = { seeMore: false }
+
+    handleClick = () => {
+        this.setState({ seeMore: true });
+    };
+
+    renderResourceComment(entity: Entity): React.Node {
+        if (!entity.resource_comment) {
+            return null;
+        }
+
+        let comment = entity.resource_comment;
+
+        return <Localized id='entitydetails-Metadata--resource_comment' attrs={ { title: true } }>
+            <Property
+                title='Resource Comment'
+                className={(this.state.seeMore) ? 'comment' : 'comment truncate'}
+            >
+                <Linkify properties={ { target: '_blank', rel: 'noopener noreferrer' } }>
+                    { comment }
+                </Linkify>
+                <button
+                    id='entitydetails-Metadata--see-more'
+                    className={(this.state.seeMore) ? 'hide' : ''}
+                    title="See More"
+                    onClick={ this.handleClick }
+                >
+                    { 'See More' }
+                </button>
```

Comment:
I don't think we'll be able to make the `<button>` localizable that way, given that it lives within the `<Localized>` tag already. @adngdb?

Reply:
Nope, the `<button>` element needs to be put inside another `<Localized>` component in order to make the content localizable. It also needs a new l10n id. Note that `<Localized>` elements are stackable, so that's not a problem.

Reply:
@adngdb I have an id of `entitydetails-Metadata--see-more` on the button now. Does this need to be different?